src/main/java/pages/DiskUsagePage.java

```
@@ -0,0 +1,108 @@
+    private WebElement uploadNewModel;
+    @FindBy(xpath = "//input[@id='model-3d-object']")
+    private WebElement uploadModel3D;
+    @FindBy(xpath = "//input[@placeholder='Model name']")
+    private WebElement modelNameField;
+    @FindBy(xpath = "//input[@placeholder='Other detail']")
+    private WebElement otherDetailsField;
+    @FindBy(id = "mui-component-select-category")
+    private WebElement categoryField;
+    @FindBy(xpath = "//li[contains(@data-value, '63494d45adb2be493ed8f49c')]")
+    private WebElement buildingOption;
+    @FindBy(id = "mui-component-select-subcategory")
+    private WebElement subcategoryField;
+    @FindBy(xpath = "//li[contains(@data-value, '634d637ffd804b0f9c8948ad')]")
+    private WebElement cabinsOption;
+    @FindBy(xpath = "//textarea[contains(@placeholder,'A few words about your client...')]")
+    private WebElement clientDescription;
+    @FindBy(xpath = "//input[@id='model-coverImage']")
+    private WebElement modelCoverImage;
+    @FindBy(xpath = "//button[contains(@type,'submit')]")
+    private WebElement saveModelButton;
+    @FindBy(xpath = "//button[normalize-space()='Done']")
+    private WebElement doneButton;
+    @FindBy(xpath = "//span[normalize-space()='My account']")
+    private WebElement myAccount;
+    @FindBy(css = "div[class='text-center']")
+    private WebElement infoAboutUsage;
+    @FindBy(xpath = "//img[contains(@alt,'Red Truck')]")
+    private WebElement truckModel;
+    @FindBy(xpath = "//button[@type='button']")
+    private WebElement deleteModelButton;
+    @FindBy(xpath = "//button[normalize-space()='Remove model']")
+    private WebElement removeModelButton;
+    @FindBy(xpath = "//span[normalize-space()='Logout']")
+    private WebElement logOutButton;
+    @FindBy(xpath = "//button[normalize-space()='Upload final model']")
+    private WebElement uploadFinalModel;
+    String pathImageFile = "/Users/mr.lee/Desktop/635e7c24643aacd5e033acfd-Peterbilt/coverImage/truck.jpg";
```

You are still using a hardcoded path from your PC. In case someone else will want to run these tests, they will fail due to the path not present in their system. We should better put these needed models in the framework itself on a resources folder/package in our repo. Thanks!

---

It makes sense. I understood now. Thank you! It's done.